server/src/main/java/org/apache/druid/server/coordinator/duty/CompactSegments.java

```diff
@@ -277,7 +300,8 @@ private CoordinatorStats doRun(
         );
         LOG.infoSegments(segmentsToCompact, "Compacting segments");
         // Count the compaction task itself + its sub tasks
-        numSubmittedTasks += findNumMaxConcurrentSubTasks(config.getTuningConfig()) + 1;
+        numSubmittedTasks++;
```

Comment:
Why do we need both numSubmittedTasks and numCompactionTasksAndSubtasks?
If we are using numSubmittedTasks as the stats, then the stat will be different. Previously, it includes subtasks and now it doesn't. Maybe this should be documented? Otherwise you might see your stat for this changed drastically.

Reply:
I think the previous one was wrong. It should count only the number of compaction tasks, not including its subtasks. I think this change would be fine since it's not documented yet. But yes, we should document it and all missing metrics.